.github/workflows/ci.yml

```diff
@@ -18,7 +42,7 @@ jobs:
       - name: Runs shell script static analysis
         run: |
           sudo apt-get install shellcheck
-          ./run-tests.sh --check-shellscript
+          ./run-tests.sh --check-shellcheck
```

Comment:
There are some inconsistencies here and in `run-tests.sh`, as sometimes we call `--check-shellcheck` and some other times `--check-shellscript`. For example, in `run_all` the function `check_shellcheck` is called, but there is no function with that name.

PS: should we maybe add also a default case in `run-tests.sh` to catch typos when calling `./run-tests.sh --check-...`? Something like this:
```bash
for arg in "$@"
do
    case $arg in
        [...]
        *) echo "invalid argument $arg" && exit 2;;
    esac
done
```

Reply:
Wanted to change the command everywhere to have the tool name <-> option name consistency, but apparently forgot some parts. Fixed.

Good idea to do catch-all rule to return error. Added.